### PR TITLE
chore: add codemod util to @compiled/eslint-plugin

### PR DESCRIPTION
**Pull request by DarkPurple141**
Hi folks! 👋 

I'm just cleaning up the old eslint rules / utils and replacing them with an alternative library I'm trying consolidate / develop to aid these sorts of rules in future.

Let me know if any objections!

### CODE REVIEW

File: packages/eslint-plugin/src/rules/jsx-pragma/index.ts [hunk 1]
```diff
@@ -134,6 +141,7 @@ export const jsxPragmaRule: Rule.RuleModule = {
           data: {
             pragma: options.runtime === 'classic' ? 'jsx' : 'jsxImportSource',
           },
+          // @ts-ignore
```
Comment (JakeLane) [inline packages/eslint-plugin/src/rules/jsx-pragma/index.ts:144]: Is this required?

File: yarn.lock [hunk 2]
```diff
@@ -52,7 +52,29 @@
   resolved "https://registry.yarnpkg.com/@babel/compat-data/-/compat-data-7.17.7.tgz#078d8b833fbbcc95286613be8c716cef2b519fa2"
   integrity sha512-p8pdE6j0a29TNGebNm7NzYZWB3xVZJBZ7XGs42uAKzQo8VQ3F0By/cQCtUEABwIqw5zo6WA4NbmxsfzADzMKnQ==
 
-"@babel/core@7.12.9", "@babel/core@^7.1.0", "@babel/core@^7.11.1", "@babel/core@^7.12.10", "@babel/core@^7.13.16", "@babel/core@^7.17.9", "@babel/core@^7.5.5", "@babel/core@^7.7.5":
+"@babel/core@7.12.9":
+  version "7.12.9"
+  resolved "https://packages.atlassian.com/api/npm/npm-remote/@babel/core/-/core-7.12.9.tgz#fd450c4ec10cdbb980e2928b7aa7a28484593fc8"
```
Comment (JakeLane) [inline yarn.lock:57]: Please change this to the `yarnpkg.com` registry

**Review by JakeLane — CHANGES_REQUESTED** [review]
Thanks for the contribution! 💯 

For the library, what's the plan for support and maintenance? My main concern is if we need to contribute changes upstream or any vulnerabilities that come up.

File: packages/eslint-plugin/src/rules/no-emotion-css/index.ts [hunk 3]
```diff
@@ -57,12 +59,14 @@ export const noEmotionCssRule: Rule.RuleModule = {
         }
       },
       ImportDeclaration(node) {
-        if (node.specifiers[0].type === 'ImportNamespaceSpecifier') {
+        if (node.specifiers[0] && isNodeOfType(node.specifiers[0], 'ImportNamespaceSpecifier')) {
```
Comment (DarkPurple141) [inline packages/eslint-plugin/src/rules/no-emotion-css/index.ts:62]: address

**Comment by itsdouges** [comment]
Closing as stale.

**Pull request closed without merging**